src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -130,6 +131,7 @@ export class SessionSubmissionPageComponent implements OnInit {
       this.regKey = queryParams.key ? queryParams.key : '';
       this.moderatedPerson = queryParams.moderatedperson ? queryParams.moderatedperson : '';
       this.previewAsPerson = queryParams.previewas ? queryParams.previewas : '';
+      this.moderatedQuestionId = queryParams.moderatedquestionId ? queryParams.moderatedquestionId : '';
```

Comment:
And call pageScrollService here. There is no need to have `moderatedQuestionId` attribute.

Reply:
we need the attribute because, here is in nginit, the scroll can only be done after ngviewinit

Reply:
I see. So we can keep `moderatedQuestionId` and call pageScrollService in the `ngOnInit` of the page component?